src/libcore/num/mod.rs

```diff
@@ -1997,27 +1997,15 @@ $EndFeature, "
 ```"),
             #[stable(feature = "rust1", since = "1.0.0")]
             #[rustc_const_stable(feature = "const_int_methods", since = "1.32.0")]
+            #[allow_internal_unstable(const_if_match)]
             #[inline]
             #[rustc_inherit_overflow_checks]
             pub const fn abs(self) -> Self {
-                // Note that the #[inline] above means that the overflow
-                // semantics of the subtraction depend on the crate we're being
-                // inlined into.
```

Comment:
This comment shouldn't be lost.
(See the original version of this code before the const hack was introduced - https://github.com/rust-lang/rust/commit/39260d9016db6925414541ad2dce6502dded3a64.)

Reply:
Understood. Re-added the comment now.